### PR TITLE
Change the scope definition

### PR DESCRIPTION
**Pull request by michel-tricot**
## What
The definition of a scope is now the bucket in which operations can happens instead of being a tuple of operations & bucket. 

for example was TYPE:CONNECTION_ID, now it is just the connection_id

## How
Change the queries

## Recommended reading order
1. `airbyte-scheduler/src/main/java/io/airbyte/scheduler/persistence/DefaultJobPersistence.java`
1. the rest

### CODE REVIEW

File: airbyte-commons/src/main/java/io/airbyte/commons/text/Sqls.java [hunk 1]
```diff
@@ -22,34 +22,20 @@
...
-  public static String getConfigId(String scope) {
-    Preconditions.checkNotNull(scope);
-
-    final String[] split = scope.split(SCOPE_DELIMITER);
-    if (split.length <= 1) {
-      return "";
-    } else {
-      return split[1];
-    }
+  public static <T extends Enum<T>> String toSqlInFragment(final Iterable<T> values) {
```
Comment (cgardens) [inline airbyte-commons/src/main/java/io/airbyte/commons/text/Sqls.java:36]: not sure i understand this method name. `iterableToSql` maybe?

**Review by cgardens — APPROVED** [review]
nice.

File: airbyte-commons/src/main/java/io/airbyte/commons/text/Sqls.java (hunk 1 above)
Comment (michel-tricot) [reply airbyte-commons/src/main/java/io/airbyte/commons/text/Sqls.java:36]: I want to say that it generate the fragment of sql that you put inside the `IN` clause

Comment (cgardens) [reply airbyte-commons/src/main/java/io/airbyte/commons/text/Sqls.java:36]: ooh. got it. okay, name is maybe okay then. i'd add a java doc comment. it's going to be hard to fully convey what you mean in a method name.

**Pull request merged**